{"uuid": "83c90458-00c0-478e-a285-189bd191af9d", "children": ["2c9116ec-2174-41d5-a154-03f82782e435", "4ed593ca-4858-458f-a495-a27a1b095f8c", "a9f47dbc-1153-4d53-b481-37d1f068b790", "3018bc35-aafa-4629-ad64-64785a76601b", "ea44b452-57b3-4768-9efe-a06634f161a8", "12e169f0-7d59-487a-981e-27641e2f07b6"], "befores": [{"name": "get_job_details", "status": "broken", "statusDetails": {"message": "requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='emia.com', port=443): Max retries exceeded with url: /jobs/ (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7faaa1bd44f0>, 'Connection to emia.com timed out. (connect timeout=None)'))\n", "trace": "  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pluggy-1.5.0-py3.10.egg/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pytest-7.4.3-py3.10.egg/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pytest-7.4.3-py3.10.egg/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/home/runner/work/scrapers_Larisa_M16/scrapers_Larisa_M16/tests/Test_Selenium_Suite/test_emia.py\", line 15, in get_job_details\n    scraper_data = emiaScraper()\n  File \"/home/runner/work/scrapers_Larisa_M16/scrapers_Larisa_M16/sites/emia_scraper.py\", line 27, in scraper\n    soup = GetStaticSoup(\"https://emia.com/jobs/\")\n  File \"/home/runner/work/scrapers_Larisa_M16/scrapers_Larisa_M16/sites/__utils/req_bs4_shorts.py\", line 36, in __new__\n    response = session.get(link, headers=headers)\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/requests-2.28.2-py3.10.egg/requests/sessions.py\", line 600, in get\n    return self.request(\"GET\", url, **kwargs)\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/requests-2.28.2-py3.10.egg/requests/sessions.py\", line 587, in request\n    resp = self.send(prep, **send_kwargs)\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/requests-2.28.2-py3.10.egg/requests/sessions.py\", line 701, in send\n    r = adapter.send(request, **kwargs)\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/requests-2.28.2-py3.10.egg/requests/adapters.py\", line 553, in send\n    raise ConnectTimeout(e, request=request)\n"}, "start": 1716355693844, "stop": 1716355823197}], "start": 1716355693844, "stop": 1716355823613}